src/doc/documentation-guide.mk2

```diff
@@ -0,0 +1,189 @@
+# Contributing Documentation in SnabbSwitch
+
+*XXX - TODO: Document different documentation targets: Casual users,
+ power users, contributors.*
```

Comment:
I think last time we talked about our Lua centric configuration format etc. we (I at least) conceded that we should probably take the boring route of using JSON as a config format. Sure we can't have comments then and the format is sub-par for the writer but at least its very well defined and we can just point to the JSON spec.

Other than that the snabbnfv `--help` got much simpler with the `PcapFilter` app since we now just point to the `filter-expression` man page.